sdk/common/src/main/java/io/opentelemetry/sdk/common/CompletableResultCode.java

```diff
@@ -136,8 +136,7 @@ public boolean isDone() {
   }
 
   /**
-   * Waits for the specified amount of time for this {@link CompletableResultCode} to complete. If
-   * it times out or is interrupted, the {@link CompletableResultCode} is failed.
+   * Waits for the specified amount of time for this {@link CompletableResultCode} to complete.
```

Comment:
I think it might be useful to also have a note in here about the fact that the timeout *doesn't* change the state of the result at all.

Reply:
Yes, I think we should add a big fat warning that after the join you need to manually check isDone/isSuccess.